docs/source/isaura.rst

```diff
@@ -1,5 +1,160 @@
+ * ``/Filters/hits_select``: flag to indicate if an event did not pass due to the lack of hits
+ * ``/Filters/topology_select``: flag that indicates whether the topology has been performed properly 
+ * ``MC info``: Monte Carlo information of events. Only if run number < 0
+
+.. _Isaura config:
+
+Config
+------
+
+Apart from the :ref:`Common arguments to every city`, the parameters to run *Isaura* are the following:
+
+.. list-table::
+   :widths: 50 100 120
+   :header-rows: 1
+
+   * - **Parameter**
+     - **Type**
+     - **Description**
+
+   * - ``vox_size``
+     - ``[float,float,float]``
+     - X, Y, and Z dimensions of the voxels used in the voxalization of the hits.
+
+   * - ``strict_vox_size``
+     - ``bool``
+     - Flag to indicate if the size of the voxels is forced to be exactly the values provided in the previous argument (*True*),
+
+       or, on the other hand, if they are allowed to change a bit for each track, aiming to optimize the voxelization process (*False*).
+
+   * - ``energy_threshold``
+     - ``float``
+     - If the energy of one of the original end-point voxels is smaller than this value,
+
+       the voxel will be dropped and its energy redistributed to the neighbours.
+
+   * - ``min_voxels``
+     - ``int``
+     - The voxel dropping procedure commented on ``energy_threshold`` can only happen if the number of voxels is larger than the value specified in this argument.
+
+   * - ``blob_radius``
+     - ``float``
+     - Radius of the blobs.
+
+   * - ``max_num_hits``
+     - ``int``
+     - Maximum number of hits for an event to be processed.
+
+
+.. _Isaura workflow:
+
+Workflow
+--------
+
+The basic idea behind Isaura is to run the "traditional" *Paolina* algorithm over the deconvoluted hits output in :doc:`beersheba`, in order to convert ``events`` into ``tracks`` with ``blobs``.
+
+The first step within the algorithm consists in checking that the number of hits is lower than the value provided in the config file (``max_num_hits``). That argument was introduced because, when running *Paolina* algorithm after :doc:`penthesilea`, there were some events that comprise such large amount of hits that the tracking information extraction took ridiculously long. As the plot presented below shows [#]_, high energy (trigger2) events usually contain around 200 *Penthesilea hits*, while there are some of them with more than 10000 hits.
+
+ .. image:: images/isaura/nhits_per_evt_r8571.jpg
+   :width: 1000
+
+The plot also shows that these events only  appear a few times within a 24h-long low-background run (around 0.05% of the total set of events). Rejecting these type of events is not a particularly worrysome issue: they would be thrown away in the posterior analysis, seeing that none of them are exclusively contained inside the fiducial volume. The spatial distributions for one of these events is presented below (where the grey dashed lines illustrate the boundaries of the chamber):
```

Comment:
`seeing that none of them are exclusively contained` --> because none of them are exclusively contained. It sounds more formal to me.